src/test/java/com/wikia/webdriver/common/remote/operations/BaseRemoteOperation.java

```diff
@@ -0,0 +1,59 @@
+package com.wikia.webdriver.common.remote.operations;
+
+import com.wikia.webdriver.common.core.Helios;
+import com.wikia.webdriver.common.core.helpers.User;
+import com.wikia.webdriver.common.logging.PageObjectLogging;
+import com.wikia.webdriver.common.remote.Discussions;
+import com.wikia.webdriver.common.remote.RemoteException;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.commons.lang3.exception.ExceptionUtils;
+import org.apache.http.HttpHeaders;
+import org.apache.http.client.methods.CloseableHttpResponse;
+import org.apache.http.client.methods.HttpEntityEnclosingRequestBase;
+import org.apache.http.entity.ContentType;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClientBuilder;
+import org.apache.http.util.EntityUtils;
+
+import java.io.IOException;
+import java.io.UnsupportedEncodingException;
+
+@AllArgsConstructor(access = AccessLevel.PACKAGE)
+public class BaseRemoteOperation {
+
+  @Getter
+  private final User user;
+
+  String execute(final HttpEntityEnclosingRequestBase requestBase) {
+    String result = StringUtils.EMPTY;
+
+    try (CloseableHttpClient client = HttpClientBuilder.create()
+        .disableContentCompression()
+        .build()) {
+      try {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Extract this nested try block into a separate method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1141)